parsers/MN.py

```diff
@@ -99,6 +101,7 @@ def fetch_production(
     # 'unknown' consists of 92.8% coal, 5.8% oil and 1.4% hydro as per 2020; sources: IEA and IRENA statistics.
     query_data["leftoverMW"] = round(
         query_data["consumptionMW"]
+        - query_data["thermalMW"]
         - query_data["importMW"]
         - query_data["solarMW"]
         - query_data["windMW"],
```

Comment:
LGTM! 
Just a quick question, should any usage/withdrawings from battery storage be included here?